mvc-framework/core/internal/components/resource-handler-registry/resourceHandler.ts

```diff
@@ -39,8 +39,6 @@ export class ResourceHandler implements Mandarine.MandarineCore.IResourceHandler
 
     public addResourceCors(cors: Mandarine.MandarineMVC.CorsMiddlewareOption): ResourceHandler {
         if(cors == undefined) return this;
-        if(cors.origin == (null || undefined)) cors.origin = Mandarine.Defaults.MandarineDefaultCorsOptions.origin;
-        if(cors.methods == (null || undefined)) cors.methods = Mandarine.Defaults.MandarineDefaultCorsOptions.methods;
```

Comment:
Based on the comment I added for the default configuration, we should keep this.